### PR TITLE
feat: Added Makefile

### PR DESCRIPTION
**Pull request by ki4070ma**
### Changes
1. Added setup.cfg
1. Added Makefile ( https://github.com/appium/python-client/issues/523 )
   * `help` command: https://marmelab.com/blog/2016/02/29/auto-documented-makefile.html
1. Fix ci.sh to set failure even when one command failed

### CODE REVIEW

File: appium/webdriver/extensions/session.py [hunk 1]
```diff
@@ -60,7 +60,7 @@ def events(self) -> Dict:
             session = self.session
             return session['events']
         except Exception as e:
-            logger.warning('Could not find events information in the session. Error:', e)
+            logger.warning('Could not find events information in the session. Error: %s', e)
```
Comment (ki4070ma) [inline appium/webdriver/extensions/session.py:63]: [Notes] Fixed pylint error

File: ci.sh [hunk 2]
```diff
@@ -1,43 +1,27 @@
...
-  echo "Please run command 'python -m isort -rc .' on your local and commit the result"
+if ! make isort ARGS=--check-only ; then
+  echo "Please run command 'make isort' on your local and commit the result"
   EXIT_STATUS=1
 fi
 
-(
-  LINT_RESULT=$(python -m pylint --rcfile .pylintrc appium test --errors-only 2>&1 | tee /dev/tty)
-  if [[ $? -ne 0 ]] ; then
-    EXIT_STATUS=1
```
Comment (ki4070ma) [inline ci.sh:19]: [Notes] This `EXIT_STATUS` scope is in subshell (`()`).

File: Makefile [hunk 3]
```diff
@@ -0,0 +1,25 @@
+.PHONY: Commands for developers
+
```
Comment (KazuCocoa) [inline Makefile:2]: How about adding `all`?

Comment (ki4070ma) [reply Makefile:2]: Added `check-all` command.

**Pull request merged**